src/Controller/GameController.php

```diff
@@ -0,0 +1,75 @@
+<?php
+namespace App\Controller;
+
+use App\Entity\Game;
+use App\Entity\Ship;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
+
+class GameController extends AbstractController
+{
+    const VERTICAL = 'VERTICAL';
```

Comment:
Those constants needs to be placed outside the controller inside a specific Class